cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/analyzer/GoalOptimizer.java

```diff
@@ -135,6 +142,13 @@ public GoalOptimizer(KafkaCruiseControlConfig config,
                                                                  overrideConfigs);
   }
 
+  /**
+   * @return if we have unfixable proposal optimization
+   */
+  private boolean hasUnfixableProposalOptimization() {
+    return _hasUnfixableProposalOptimization;
```

Comment:
why do we need such private accessor for the _hasUnfixableProposalOptimization?

Reply:
i have seen sensors access the variables through a function in Cruise Control package. Hence i followed the same standard. Also in the past i have tried referencing the variable and it did not work well. I dont know the exact reason why that is the case though.